src/toxcore/dht/dht_friend.rs

```diff
@@ -89,9 +89,10 @@ impl DhtFriend {
 
         let mut peers_cache = server.get_peers_cache().write();
 
-        let nodes_sender = bootstrap_nodes.nodes.iter()
+        let bootstrap_nodes = bootstrap_nodes.to_packed_node();
+        let nodes_sender = bootstrap_nodes.iter()
             .map(|node| {
-                let client = peers_cache.entry(node.pk).or_insert_with(ClientData::new);
+                let client = peers_cache.entry(node.pk).or_insert_with(|| DhtNode::new(*node));
```

Comment:
Use just `or_insert` method in such cases, no need to wrap it into closure.

Reply:
```
let client = peers_cache.entry(node.pk).or_insert_with(DhtNode::new(*node));
    the trait `std::ops::FnOnce<()>` is not implemented for `toxcore::dht::dht_node::DhtNode

Reply:
Not `or_insert_with` but [`or_insert`](https://doc.rust-lang.org/std/collections/hash_map/enum.Entry.html#method.or_insert).

Reply:
done